CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 * Python, Node: Added TYPE command ([#945](https://github.com/aws/glide-for-redis/pull/945), [#980](https://github.com/aws/glide-for-redis/pull/980))
 * Python, Node: Added HLEN command ([#944](https://github.com/aws/glide-for-redis/pull/944), [#981](https://github.com/aws/glide-for-redis/pull/981))
 * Node: Added ZCOUNT command ([#909](https://github.com/aws/glide-for-redis/pull/909))
-* Python: Added ECHO command ([#953](https://github.com/aws/glide-for-redis/pull/953))
+* Python, Node: Added ECHO command ([#953](https://github.com/aws/glide-for-redis/pull/953), [#1010](https://github.com/aws/glide-for-redis/pull/1010))
```

Comment:
this isn't added to the 0.2.0 release. you need to add this change to the next release.